src/databricks/labs/ucx/install.py

```diff
@@ -812,6 +812,26 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def repair_run(self, workflow):
+        try:
+            job_id = self._state.jobs.get(workflow)
+            if not job_id:
+                logger.warning(f"{workflow} job does not exists hence skipping Repair Run")
```

Comment:
Immediately return from function and de-dent the next block of code

Reply:
Changed according